cloud-init/write-aws-config.tpl.sh

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env bash
+# This is a Terraform template file, and the input variables are
+# passed in via templatefile().
+#
+# shellcheck disable=SC2154
+
+# This script is used write out an AWS configuration to the home
+# directory of a specified user.  The AWS configuration gives the user
+# access to the Terraformer, organization read-only, and
+# cisagov/cool-assessment-terraform Terraform state read-only roles.
+
+# Input variables are:
+# * aws_region - the AWS region where the roles are to be assumed
+# * permissions - the octal permissions to assign the AWS
+#   configuration
+# * read_cool_assessment_terraform_state_role_arn - the ARN of the
+#   IAM role that can be assumed to read the Terraform state of the
+#   cisagov/cool-assessment-terraform root module
+# * organization_read_role_arn - the ARN of the IAM role that can be
+#   assumed to read information about the AWS Organization to which
+#   the assessment environment belongs
+# * redirector_route53_role_arn - the ARN of the IAM role that can be
+#   assumed to create, delete, and modify Route53 records in the RTA
+#   redirector account
```

Comment:
Is it worth generalizing this variable so that it is not so explicitly tied to RTA and their redirector account?  The answer may be "no", but I wanted to make sure we at least discussed it.  I don't have a great name suggestion, but I was thinking something along the lines of `assessor_owned_route53_role_arn` - that can be a starting point - I'm sure we can do better.

Reply:
Really I'm just creating the profile.  If you have a role from any other account for which you have delegated authority to this assessment account then you could put it there.  The RTA folks just happen to need to give us a role to allow them to make Route53 changes in their own account.  In that sense the profile is very general and I could just as well call it `role_arn`.  We won't know or care what they use the profile to do.

That said, I hadn't really envisioned any other uses for this role.  I guess I'm fine with changing the name if you and @mcdonnnj can decide on what you want.

Reply:
The general nature of this role makes it tough to name, but I suppose as long as we document it well enough then that won't matter.  I'd be fine with explicitly referencing this RTA/Route53 use case in the variable description.  What do you think about `assessment_team_delegated_account_role_arn`?

Reply:
I ended up going with `assessor_account_role_arn` as I felt the rest was redundant.  See commit 5b34a8a.

Reply:
`assessor_account_role_arn` works for me - thanks!